examples/source/visual-effects/AMP_Story_Animations.html

```diff
@@ -8,6 +8,8 @@
   ## Introduction
 
   AMP Stories support animations out-of-the-box.  Every element inside an `amp-story-page` can define an entrance animation. You can find a list of all supported animations [here](/content/amp-dev/documentation/components/reference/amp-story.md#animations).
+
+  [tip type="note"]Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.[/tip]
```

Comment:
Can we add a link to https://github.com/ampproject/amphtml/issues/24821, which tracks respecting the `prefers-reduced-motion` directive for such users?